mix.exs

```diff
@@ -31,11 +31,11 @@ defmodule RailwayIpc.MixProject do
       {:ecto_sql, "~> 3.0"},
       {:elixir_uuid, "~> 1.2"},
       {:ex_doc, ">= 0.0.0", only: :dev, runtime: false},
-      {:ex_machina, "~> 2.3", only: :test},
+      {:ex_machina, "~> 2.3", only: [:test, :e2e]},
       {:google_protos, "~> 0.1"},
       {:jason, "~> 1.1"},
       {:mix_test_watch, "~> 0.8", only: :dev, runtime: false},
-      {:mox, "~> 0.5", only: :test},
+      {:mox, "~> 0.5", only: [:test, :e2e]},
```

Comment:
Unfortunately we need to add mox to the `e2e` environment even though it's not used. The test support code won't compile w/o it.